projects/dsp-ui/src/lib/action/pipes/array-transformation/sort-by.pipe.ts

```diff
@@ -0,0 +1,29 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+    name: 'dspSortBy'
+})
+export class SortByPipe implements PipeTransform {
+
+    /**
+     * compares value by value and sorts by alphabetical order
+     */
+    transform(array: Array<any>, args: string): Array<any> {
```

Comment:
see my comment above about sorting

Reply:
can also be moved to a service

Reply:
@mdelez in order to avoid breaking changes, you should mark the pipe as deprecated and indicate that the service is the new way to go

Reply:
moved to a new service called `sorting.service` in 63afcde and marked the old pipe as deprecated